evap/results/templates/distribution_bar.html

```diff
@@ -1,11 +1,38 @@
 {% load results_templatetags %}
+{% load evaluation_filters %}
 
 {% spaceless %}
-<div class="distribution-bar">
-    {% for answer, value in distribution.items %}
-        {% if value != 0 %}
-            <div class="vote-bg-{{ answer }}{% if participants_warning %} participants-warning{% endif %}" style="width: {% widthratio value distribution.values|sum 100 %}%;">&nbsp;</div>
+{% if average %}
+    {% if show_participant_count %}
+        <div class="grade-bg-result-bar-count text-center{% if participants_warning %} participants-warning{% endif %}">
+            <span class="fas fa-user small"></span> {{ result.total_count }}
+        </div>
+    {% endif %}
+    <div class="d-flex">
+        <div class="distribution-bar">
+            {% for answer, value in distribution.items %}
+                {% if value != 0 %}
+                    <div class="vote-bg-{{ answer }}{% if participants_warning %} participants-warning{% endif %}" style="width: {% widthratio value distribution.values|sum 100 %}%;">&nbsp;</div>
+                {% endif %}
+            {% endfor %}
+        </div>
+        <div class="grade-bg-result-bar text-center{% if participants_warning %} participants-warning{% endif %}" style="background-color: {{ average|gradecolor }}">
+            {% if result.question.is_yes_no_question %}
+                {{ result.approval_count|percentage:result.total_count }}
+            {% else %}
+                {{ average|floatformat:1 }}
+            {% endif %}
+        </div>
+    </div>
+{% else %}
+    <div class="d-flex" data-toggle="tooltip" data-placement="left" title="{% trans 'Not enough answers were given.' %}">
+        {% if show_participant_count %}
+            <div class="grade-bg-result-bar-count text-center grade-bg-disabled">
+                <span class="fas fa-user small"></span> {{ result.total_count }}
+            </div>
         {% endif %}
```

Comment:
same here. i think it's fair to not give the participant count the tooltip, since the participant count itself has no problem with the low number of answers.

Reply:
what is the same as where?

Reply:
i had some struggles when i was explicitly trying to get the tooltip around everything including the participant count so that it doesn't overlay the information. i agree that the participant count itself doesn't need the information but i don't want to hide it when the information is shown.

Reply:
what about any of the other three directions where the tooltip can be placed?

Reply:
we don't display the tooltips to the right because we can't assume there is enough space. on the top and bottom other questions/courses would be overlapped which is also not really nice.
what's the problem with showing it left of the participation again?

Reply:
my primary problem is that large portions of this file are not used by all of it's callers, and there are too many code paths through this file, making this file hard to understand. moving some parts that are only used by one caller to that caller was a suggestion to help with that. if you have other solutions, i'm fine with that as well.

Reply:
alright. the idea was to define all the elements only once and everywhere just using what is needed in the specific situation. i wanted to avoid defining similar elements that only differ in small details more than once. i'll see if i can rewrite the result_bar and distribution_bar in general.